modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -938,7 +938,43 @@ definition {
 	@priority = 4
 	test DisplayDueDateWithHyphenWhenSLANotStarted {
 
-		// TODO LRQA-67729 DisplayDueDateWithHyphenWhenSLANotStarted pending implementation
+		WorkflowMetrics.goToWorkflowSLASettings(workflowProcessName = "Single Approver");
```

Comment:
Just added the test that was written from [this PR](https://github.com/rodrigocunhaa/liferay-portal/pull/551). It has been tested and passes locally.